rotkehlchen/chain/evm/accounting/aggregator.py

```diff
@@ -157,6 +153,73 @@ def get_accounting_settings(self, pot: 'AccountingPot') -> dict[str, TxEventSett
         )
         return result
 
+    def _get_settings_for_customization(self, pot: 'AccountingPot') -> dict[str, TxEventSettings]:
```

Comment:
nitpick: Both in the name and in the docstring don't use customization as a word. Since that we should use for customizing the tax treatment of each different type of event when https://github.com/rotki/rotki/issues/4341 is done.

Perhaps call it editable_combinations and in the docstrings explain users can edit the transactions type/subtype to get them.